_posts/static/2018-02-01-installation-wizard.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+howtos: true
+published: true
+title: Installation v2.0.0
+permalink: howtos/installation-wizard
+description: Now with Wizard
+---
+
+## Server Recommendations
+
+You need several servers to run Flyve MDM:
+
+* a server running Linux, Apache, Mysql/MariaDB and PHP (a LAMP server) for the backend (GLPI and Flyve MDM for GLPI),
+* a server running Mosquitto,
+* a server running the web interface. It may run on the same server as GLPI
+
+<img src="{{ '/images/picto-information.png' | absolute_url }}" alt="Good to know:" height="16px"> This is not mandatory, Mosquitto and GLPI can run in the same server, there is no limitation.
+
+## Flyve MDM Overview
+
+Flyve MDM v2.0.0 runs on GLPI 9.2 or later. It depends on inventory features of FusionInventory for GLPI, therefore it is required FusionInventory 9.2+1.0 or later. Check the compatibility matrix to know which versions are appropriate.
+
+### Compatibility matrix
+
+<table>
+    <tr>
+        <td style="width:150px">GLPI</td>
+        <td style="width:100px">9.1.1</td>
+        <td style="width:100px">9.1.2</td>
+        <td style="width:100px">9.1.3</td>
+        <td style="width:100px">9.2.0</td>
```

Comment:
9.2.1 required for the development version and the incoming release

The  text above the chart may need an update if you want to refer to the next release.